plugin/src/androidxr/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!-- Permissions needed by OpenXR -->
+    <uses-permission android:name="org.khronos.openxr.permission.OPENXR" />
+    <uses-permission android:name="org.khronos.openxr.permission.OPENXR_SYSTEM" />
+
+    <queries>
+        <intent>
+            <action android:name="org.khronos.openxr.OpenXRRuntimeService"/>
+        </intent>
```

Comment:
Should this include the other `<intent>` tag that was recently added to the khronos `AndroidManifest.xml`?

```
        <intent>
            <action android:name="org.khronos.openxr.OpenXRApiLayerService" />
        </intent>
```

Reply:
I'll try to do as we discussed and add a dependency on the khronos aar library so we can get those automatically via manifest merging.